source/user-manual/capabilities/vulnerability-detection/FAQ.rst

```diff
@@ -0,0 +1,86 @@
+.. Copyright (C) 2015, Wazuh, Inc.
+
+.. meta::
+   :description: Learn how to solve common issues with the Vulnerability Detection module in this section of the documentation.
+
+Frequently Asked Questions
+==========================
+
+This guide presents the solution to common problems that might occur when using the Vulnerability Detection module.
+These issues are related to the configuration, installation, and operation of the module and aren't known issues that require an upgrade to be fixed.
+
+
+Communication problem between the Wazuh manager and the Wazuh Indexer
+---------------------------------------------------------------------
+
+.. list-table:: Issue description
+   :widths: 15 50
+
+   * - Problem
+     - There are no vulnerabilities reported in Wazuh Dashboard or the `wazuh-states-vulnerabilities-*` index isn't created. It's also possible to find messages like `IndexerConnector initialization failed for index 'X', retrying until the connection is successful` in the manager's logs.
+   * - Cause
+     - A misconfiguration of the vulnerability detector, indexer connector or wazuh-keystore components is the most common cause.
+
+
+Known configuration/environment problems
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+- Make sure that the wazuh-keystore has been properly updated with the right admin credentials (username and password). See  :ref:`wazuh-keystore <wazuh-keystore>` for more details.
+- Verify the name and location of the certificates configured under the indexer connector section in the manager's configuration file. Also, the host IP/FQDN must match the one used for the certificate creation.
+
+The related configuration block is the following, the `<host>` and `<ssl>` sections misconfiguration can cause the connection problem:
+
+.. code-block:: xml
+   :emphasize-lines: 4,8,10,11
+
+   <indexer>
+     <enabled>yes</enabled>
+     <hosts>
+       <host>https://0.0.0.0:9200</host>
+     </hosts>
+     <ssl>
+       <certificate_authorities>
+         <ca>/etc/filebeat/certs/root-ca.pem</ca>
+       </certificate_authorities>
+       <certificate>/etc/filebeat/certs/filebeat.pem</certificate>
+       <key>/etc/filebeat/certs/filebeat-key.pem</key>
+     </ssl>
+   </indexer>
+
+
+The connection with the Wazuh Indexer can be verified with a `curl` command from the Wazuh manager. Replace the credentials and certificate paths with the ones from the configuration:
+
+.. code-block:: console
+
+    # curl --cacert <ROOT_CA> --cert CERTIFICATE_PEM<> --key <CERTIFICATE_KEY> -u <USER>:<PASS> -XGET https://<INDEXER_IP>:9200/_cluster/health
+
+If this command isn't successful, the vulnerability detector module won't be able to connect to the Indexer either.
+An alternative is to skip the certificate verification with the `-k` option, this can help to identify if the problem is related to the certificates:
```

Comment:
```suggestion
An alternative is to skip the certificate verification with the `-k` option, this can help to identify if the problem is related to the certificates:
```

Why is this an alternative? I know what you refer but maybe a user won't

We could says something in the line of "To verify the certificates are the problem and not the indexer, skip the certificate verification ..."

Reply:
I will propose a fix.